shining-pearl-kazza-palkia-glitchless/Chimchar-Palkia-tracker.mdr

```diff
@@ -0,0 +1,37 @@
+:::tracker{species="Chimchar" baseStats="[[44, 58, 44, 58, 44, 61], [64, 78, 52, 78, 52, 81]]" type="Fire" generation=8}

```

Comment:
Remove this file, since it's already covered by the routefile.

Reply:
Fixed both i think